docs/src/whatsnew/latest.rst

```diff
@@ -56,6 +56,11 @@ This document explains the changes made to Iris for this release
    cubes with masked :class:`iris.coords.CellMeasure`.
    (:issue:`5147`, :pull:`5181`)
 
+#. `@scottrobinson02`_ fixed :class:`iris.util.new_axis` creating an anonymous new
```

Comment:
@scottrobinson02 As you're not a core developer (just yet :wink:) you need to add a link to your user at the bottom of the [latest.rst here](https://github.com/SciTools/iris/blob/main/docs/src/whatsnew/latest.rst?plain=1#L136-L137)